ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -713,6 +713,30 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
         )
       }
     }
+
+    @Explanation(
+      """|A user can have only a limited number of user rights.
+                    |There was an attempt to create a user with too many rights or grant too many rights to a user."""
+    )
+    @Resolution(
+      """|Retry with a smaller number of rights or delete some of the already existing rights of this user.
+                   |Contact the participant operator if the limit is too low."""
+    )
+    object TooManyUserRights
+        extends ErrorCode(
+          id = "TOO_MANY_USER_RIGHTS",
+          ErrorCategory.InvalidGivenCurrentSystemStateOther,
+        ) {
+      case class Reject(_operation: String, userId: String)(implicit
+          loggingContext: ContextualizedErrorLogger
+      ) extends LoggingTransactionErrorImpl(
+            cause = s"cannot ${_operation}, as user \"${userId}\" would have too many rights."
```

Comment:
This formulation assumes that the clients pass the operation text that can be interpreted as a verb or a verb phrase. This can be easily broken inadvertently. I suggest saying 
"${_operation} failed, as user \"${userId}\" would have too many rights."

Reply:
I'm a bit reluctant to change this as this is the wording we are already using for two other errors (`UserNotFound` and `UserAlreadyExists`). Could do a small separate PR to change all three consistently if you wish.

Reply:
@pbatko-da : feel free to change it in this PR for all of them. Better are messages are good to have, and givne that we don't change the error-id it is also a change that's always OK to do. Moreover, these two errors have been introduced as part of the same line of work that you are currently working on.

Reply:
I noticed that we don't have any tests that would assert what these messages exactly contain.
I'd say testing it in test-tool is inappropriate because the exact message wording is not a public API.
The other option is creating a unit test for `com.daml.platform.apiserver.services.admin.ApiUserManagementService`. 
Wdyt?

Reply:
I would not test this as unit test, but instead make sure that you've seen an example message and are OK with it.

At least from where I'm sitting looking at the actual log output and thinking of how to make it better provides more value than enshrining the current messages into a unit test. @mziolekda what do you think?

Reply:
agree with @meiersi-da